chromadb/db/index/hnswlib.py

```diff
@@ -244,7 +253,7 @@ def get_nearest_neighbors(self, query, k, ids=None):
 
         s2 = time.time()
         # get ids from uuids as a set, if they are available
-        labels = {}
+        labels: Set[int] = set()
```

Comment:
Just to clarify: this was always intended to be a set, and initializing to a dict was just wrong?

Reply:
Yeah its only used as a set below on line 258and the init was wrong